installer/bundle/bundle_skel_Linux.sh

```diff
@@ -616,11 +616,9 @@ case "$installMode" in
     U)
         echo "Updating cross-platform agent ..."
         if [ $PROVIDER_ONLY -eq 0 ]; then
+	    shouldInstall_omi
+	    pkg_upd_list $OMI_PKG omi $?
             check_if_pkg_is_installed omi
```

Comment:
Yes, line 621 is useless- I will remove it. sorry. But the indentation seems fine to me..  perhaps I accidentally used  a tab instead of space. I will fix it.

To repro the issue - we need to install the old version (1.5.*).  Then get this shell bundle and run with ```--upgrade``` option.
Without the fix, it fails complaining omi is missing. With the above fix it upgrades to the correct version. (The same is true for earlier iteration).